HelperApp/src/test/java/com/cdw/springenablement/helper_App/service/UserServiceImplTest.java

```diff
@@ -132,18 +132,22 @@ public void testUpdateHelperSpecialization_Success() throws Exception {
     @Test
     public void testBookTechnician_NonResidentBook() {
         BookingTechnicianDto bookingTechnicianDto = new BookingTechnicianDto();
-        bookingTechnicianDto.setTimeSlotId(4);
-        bookingTechnicianDto.setHelperId(1);
-        bookingTechnicianDto.setUserId(1);
+        long timeSlotID=4L;
```

Comment:
In Test cases try to utilize commonized values of datatype and also cover negative flows